core/shared/src/main/scala/org/polyvariant/colorize.scala

```diff
@@ -121,12 +150,43 @@ object colorize {
 
   private[colorize] case class CurrentColor(value: String) extends AnyVal
 
+  sealed trait ColorMode
+
+  object ColorMode {
+    private[colorize] case object Ansi extends ColorMode
+    private[colorize] case object TrueColor extends ColorMode
+  }
+
   final case class RenderConfig(
-    resetString: String
+    mode: ColorMode,
+    resetString: String,
   )
 
   object RenderConfig {
-    val Default: RenderConfig = RenderConfig(resetString = Console.RESET)
+    private val TrueColorFlags = Set("truecolor", "24bit")
+
+    val Default: RenderConfig = RenderConfig(mode = ColorMode.Ansi, resetString = Console.RESET)
```

Comment:
Should `Default` detect the color mode automatically? While it makes sense, it can lead to non-obvious differences between local and CI environments.

Reply:
I think we shouldn't do it fully automatically.

How about this: we default to Ansi, but allow users to create their own `colorize` object which contains the given configuration. Then we don't need `renderConfigured` to be public, and there's only a single place where you have to configure the rendering.

Reply:
so in this case, the current `object colorize` would be sth like `object colorize extends Colorize(ANSI, Console.RESET)`, effectively removing the need for `RenderConfig`.